OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+the columns `MedianAdjustedNPX` and `QSNormalizedNPX`. For each assay a
+recommendation is listed in the `BridgingRecommendation` column and
+lists what method, if any should be used for that assay. Additional
+columns including `OlinkID_HT` and `OlinkID_3072` map the assays across
+products and the `Project` column lists the name of the project based on
+the `exploreht_name` and `explore3072_name` arguments. The resulting
+data set will contain the newly bridged Explore 3072 data set. The
+reference Explore HT data will be concatenated to the Explore 3072 data.
+As the reference data is not altered during normalization, the
+normalized NPX values in the Explore HT data will be the same as the
+values in the NPX column which contains the non-normalized data.
+
+```{r, post_bridging_3k_data , echo=FALSE, eval=TRUE}
+#npx_br_data |> 
+OlinkAnalyze::npx_data1 |> 
+  dplyr::filter(SampleID == OlinkAnalyze::npx_data1$SampleID[1]) |> 
+  head(10) |> 
+  dplyr::select(SampleID, OlinkID, NPX) |>
+  dplyr::mutate(OlinkID_E3072 = c("OID20117",
+                                  "OID31162",
+                                  "OID30796",
+                                  "OID20054",
+                                  "OID20118",
+                                  "OID31163",
+                                  "OID30799",
+                                  "OID20050",
+                                  "OID30789",
+                                  "OID20040"))|>
+  dplyr::mutate(MedianCenteredNPX = NPX + 1,
+         QSNormalizedNPX = NPX + 1.5,
+         Project = "new",
+         BridgingRecommendation = c("Not Bridgeable",
+                                    "MedianCentering",
+                                    "QuantileSmoothing",
+                                    "MedianCentering",
+                                    "QuantileSmoothing",
+                                    "Not Bridgeable",
+                                    "MedianCentering",
+                                    "QuantileSmoothing",
+                                    "MedianCentering",
+                                    "QuantileSmoothing")) |> 
+  kableExtra::kbl(booktabs = TRUE,
+      digits = 1,
+      caption = "Table 4. First 10 rows of combined datasets after bridging.") |>
+  kableExtra::kable_styling(bootstrap_options = "striped", full_width = FALSE, font_size = 10, 
+                position = "center", latex_options = "HOLD_position") |> 
+  kableExtra::scroll_box(width = "100%")
+```
+
+## Evaluating the quality of bridging
+
+PCA is used to assess the quality of bridging by determining if the
+sample controls (SCs) and bridging samples appear closer after bridging.
+Two PCAs can be generated, one containing the SCs and one containing the
+bridging samples. Prior to bridging there will be a noticeable
+separation between products which should decrease after bridging.
+
+```{r pca2, message=FALSE, fig.cap= f8, eval=FALSE, echo = TRUE}
+## After bridging
+
```

Comment:
Should all filter and mutate steps include "dplyr::" for consistency with other lines?